### PR TITLE
Add factory world

### PR DESCRIPTION
**Pull request by marioney**
The Workcell from the [hrwros_support](https://github.com/rosin-project/mooc_hrwros_code/tree/melodic-devel/hrwros_support/urdf/workcell) package is added with some slight changes to make room for the ridgeback robot to move.

The map for this was also created on https://github.com/rosin-project/metacontrol_nav/commit/8f57deac1e2f94a67ba1e3ac827c163b59cf961b and need to be updated

### CODE REVIEW

File: launch/MVP_metacontrol_world.launch [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
   <arg name="rviz" default="true" />
   <arg name="use_sim_time" default="true" />
-  <arg name="gui" default="false" />
+  <arg name="gui" default="true" />
```
Comment (marioney) [inline launch/MVP_metacontrol_world.launch:5]: This change should not be here

Comment (marioney) [reply launch/MVP_metacontrol_world.launch]: Ok, changed back to false

**Comment by chcorbato** [comment]
@marioney what is the launch file intended for the experiments in the factory? Shall I try it?

**Comment by marioney** [comment]
> @marioney what is the launch file intended for the experiments in the factory? Shall I try it?

It's the same as before, the MVP_metacontroller_world.launch.   Please go ahead and try it.

**Comment by chcorbato** [comment]
With
`roslaunch metacontrol_sim ridgeback_world.launch`

I get this error:
```
[FATAL] [1589442813.392409612, 2761.797000000]: Failed to start controller in realtime loop. This should never happen.
```

**Comment by marioney** [comment]
>  get this error:
> 
> ```
> [FATAL] [1589442813.392409612, 2761.797000000]: Failed to start controller in realtime loop. This should never happen.
> ```

That's strange.  According to [this answer](https://answers.ros.org/question/347779/failed-to-start-controller-in-realtime-loop-with-urure-robot-driver/?answer=347842#post-id-347842) it may have to do with an update of the controllers.

Please do
```
 sudo apt-get --only-upgrade install ros-*
```
And rebuild the ws.

**Comment by chcorbato** [comment]
> > get this error:
> > ```
> > [FATAL] [1589442813.392409612, 2761.797000000]: Failed to start controller in realtime loop. This should never happen.
> > ```
> 
> That's strange. According to [this answer](https://answers.ros.org/question/347779/failed-to-start-controller-in-realtime-loop-with-urure-robot-driver/?answer=347842#post-id-347842) it may have to do with an update of the controllers.
> 
> Please do
> 
> ```
>  sudo apt-get --only-upgrade install ros-*
> ```
> 
> And rebuild the ws.

This fixed the problem indeed, thanks!

> It's the same as before, the MVP_metacontroller_world.launch. Please go ahead and try it.

Awesome! working like a charm

**Pull request merged**